jobs/rabbitmq-server/spec

```diff
@@ -18,6 +18,8 @@ properties:
   rabbitmq-server.ssl.security_options:
     description: "SSL security options (currently only 'enable_tls1_0')"
     default: []
+  rabbitmq-server.ssl.depth:
```

Comment:
To be consistent with the naming from upstream project we should name this property `rabbitmq-server.ssl.verification_depth`